### DISCUSSION TITLE
About the description of the Downloading and Configuring document

### DISCUSSION THREAD

**Discussion by nsano-rururu** [jertel/elastalert2#394] [Ideas]
For elastalert2, the version of elasticsearch-py is fixed at 7.0.0.
elasticsearch 7.x should be fine, but I think it would be better to mention that non-7.x versions should reinstall the version that supports connection with elasticsearch-py.
It's unconfirmed, but I think that elasticsearch-py 7.0.0 and elasticsearch 6.x also have no problem connecting.

**curret**

Elasticsearch 5.0+:

```
$ pip install "elasticsearch>=5.0.0"
```

**after**

Elasticsearch 6.x

```
$ pip uninstall elasticsearch
$ pin install elasticsearch>=6.0.0,<7.0.0
```

Elasticsearch5.x

```
$ pip uninstall elasticsearch
$ elasticsearch>=5.0.0,<6.0.0
```

Since Elasticsearch 5.0 is EOL, I think there is an option not to describe it.
After that, I thought it would be better to write somewhere in the documentation which version of Elasticsearch is supported.

**Comment by jertel** [comment]
Yes, I agree with dropping support for ES 5.

**Reply by nsano-rururu** [reply]
thank you for your answer.
Then, after confirming that there seems to be no problem even if you check the operation with the latest version of Elasticsearch 6, issue a pull request to delete the following description from the document.

Elasticsearch 5.0+:

```
$ pip install "elasticsearch>=5.0.0"
```

**Reply by nsano-rururu** [reply]
When elasticsearch 8 is released, I will add the explanation for elasticearch 6.x or 7.x.
Also, I think you need to prepare docker images for elasticearch8 (pip install elasticsearch == 8.0.0) and elasticearch6 / 7 (pip install elasticsearch == 7.0.0) separately.
